src/azul/drs.py

```diff
@@ -120,20 +120,25 @@ def encode_access_id(token_str: str) -> str:
     """
     Encode the token as an access id in a URL safe way.
 
-    >>> encode_access_id('tedeschi trucks')
-    'dGVkZXNjaGkgdHJ1Y2tz'
+    Notice that base64 padding is stripped off
```

Comment:
This comment sits atop an example using `urlsafe_b64encode` so either it needs more details to explain that the following example does not strip padding but that `encode_access_id` does,  or be moved so it sits above the `encode_access_id` example.
```suggestion
    Notice that with `encode_access_id` base64 padding is stripped off
```

Reply:
Good point.